client/src/App.js

```diff
@@ -1,19 +1,28 @@
 import React from "react";
-import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
+import { BrowserRouter, Route, Redirect, Switch } from "react-router-dom";
 
-import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+import AuthPage from "./pages/AuthPage";
 
 import "./App.css";
 
+//TODO: Implement this
+function browserHasToken() {
+  return false;
+}
+
 function App() {
   return (
-    <MuiThemeProvider theme={theme}>
```

Comment:
are you planning on not using themes at all?

Reply:
Yeah, I think we can probably do without using a theme provider like this. We can still use the theme.js file as a central record for style related constants like colors, typography, rhythm, etc.